ConnectedCanteens/ngsild-payloads/Restaurants/restaurantCollery.json

```diff
@@ -16,7 +16,7 @@
             "observedAt": "2020-06-30T14:30:00Z",
             "observedBy": {
                 "type": "Relationship",
-                "object": "urn:ngsi-ld:Device:Concentrator01"
+                "object": "urn:ngsi-ld:EdgeGateway:01"
```

Comment:
on this property instance, we could also have the state (`Measured`?) and audience (`All`?)

Reply:
I think so, added [here](https://github.com/easy-global-market/saint-quentin-data-models/pull/21/commits/eb0948ffdb691d14c003ffd67a7654eb2083c4d1).